### DISCUSSION TITLE
Resettare home assistant hass.io

### DISCUSSION THREAD

**Discussion by francescobuonfigliodomotica** [indomus/forum#608] [HUB: Home Assistant]
Ciao, ho necessità di resettare hass.io ma non trovo una guida. Sapete darmi indicazioni?
Grazie, Francesco

**Comment by snakuzzo** [comment]
Rimuovi il file del db, il `configuration.yaml`, la directory `.storage` e riavvii.
In alternativa riflashi hassos

**Reply by francescobuonfigliodomotica** [reply]
ok snakuzzo, grazie mille. Per riflashare hassos basta sovrascrivere rifacendo la procedura?

**Reply by snakuzzo** [reply]
sì
